src/components/Avatar/index.tsx

```diff
@@ -13,23 +16,68 @@ interface AvatarProps {
   borderColor?: string;
 }
 
-export const Avatar = ({ source, size, borderColor }: AvatarProps) => (
-  <div className="Avatar">
-    {source.map((item) => {
-      const customStyle = {
-        ...(borderColor ? { "--Avatar-border-color": borderColor } : {}),
-        ...(size ? { "--Avatar-size": size } : {}),
-        ...(item.backgroundColor
-          ? { "--Avatar-background-color": item.backgroundColor }
-          : {}),
-        ...(item.isFullSizeImage ? { "--Avatar-image-size": "100%" } : {}),
-      } as React.CSSProperties;
-
-      return (
-        <div key={item.altText} className="Avatar__item" style={customStyle}>
-          <img alt={item.altText} src={item.iconUrl} />
-        </div>
-      );
-    })}
-  </div>
-);
+const getCodeAndKey = (asset: string) => {
+  const splitArr = asset.split(":");
+
+  return {
+    assetCode: splitArr[0],
+    issuerKey: splitArr[1],
+  };
+};
+
+const nativeAsset = {
+  assetCode: "XLM",
+  iconUrl: StellarLogo,
+};
+
+export const Avatar = ({ source, size, borderColor }: AvatarProps) => {
```

Comment:
Maybe we could create a new component (`AssetAvatar`, for example) that uses `Avatar` component internally? That way we could move the `Avatar` to the design system.